src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/custom.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 # pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
```

Comment:
Can you mark this disable statement to the function where it is needed? It is not encouraged to disable a pylint rule for the scope of the entire file.